.github/workflows/client.yaml

```diff
@@ -12,23 +12,25 @@ jobs:
 
     steps:
     - name: Checkout repository
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
 
     - name: Cache Node modules
-      uses: actions/cache@v2
+      uses: actions/cache@v3
       with:
         path: ~/.npm
         key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
```

Comment:
You still have all the old (and I think non-functional) caching, but aren't using the new `npm` caching command. I would definitely fix that as I think it will speed things up for the students.

Reply:
Can you help -- is this a comment on line 21? This matches what we have in the iteration database, and I thought we were using the newest one there.